wocky/wocky-tls.c

```diff
@@ -50,14 +50,12 @@
 
 #include "wocky-tls.h"
 
-#include <gnutls/x509.h>
-#include <gnutls/openpgp.h>
-
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #include <dirent.h>
 
+/* DANWFIXME: allow configuring compression options */
```

Comment:
What about this?

Reply:
Part of original patch :)

Reply:
But ok, perhaps we can replace that with more valuable comment like - use GIO_USE_TLS and G_TLS_GNUTLS_PRIORITY to set the SSL options

Reply:
I will test gio-tls for sailfish the next days.

Reply:
Sailfish SDK doesn't build this branch due to unknown types and functions in test-resolver.c 
like _lookup_by_name_with_flags_async_ which is available since glib 2.60, Sailfish is still on glib 2.56. I'm not sure if it only affects tests, though.

Reply:
Should be 2.59 according to tags, but can you drop the last commit then? that's where I've made this addition as all test were failing on arch due to this missing call.

Reply:
Wocky compiles when dropping the last commit, but gabble-0.18 doesn't compile with wocky-master. Sailfish needs gabble-0.18 due to an old telepathy-glib :-/ 

So I guess I'll stick to openssl for my Sailfish build, that shouldn't be a hindrance for wocky-master development ...

Reply:
Ok, let me post my current testing branch 
https://github.com/rufferson/telepathy-gabble/tree/test-merged
which includes your and mine PRs as well as other minor cleanup patches I've cherry-picked (including GIO-TLS one)

Reply:
I see, when including _Adjust callbacks to Wocky GIO-TLS changes_ gabble-18.4 builds with gio-tls on Sailfish SDK. I will take a deeper look these days.

Reply:
We need to apply the callback patch to gabble master, don't we?

Reply:
when we update submodule tip, yes